internal/cli/root.go

```diff
@@ -59,6 +59,10 @@ func Execute() {
 	viper.BindPFlag("no-colors", pf.Lookup("no-colors"))
 	argToEnv["colors"] = "EPINIO_COLORS"
 
+	pf.StringP("magic-domain", "", "omg.howdoi.website", "Magic domain to use self-signed certs with")
+	viper.BindPFlag("magic-domain", pf.Lookup("magic-domain"))
+	argToEnv["magic-domain"] = "EPINIO_MAGIC_DOMAIN"
```

Comment:
This makes `--magic-domain` a global flag. If you installed Epinio with `--magic-domain=nip.io` and you started the epinio server with no value (in which case the default `omg.howdoi.website` will be used, this line will have a problem: https://github.com/epinio/epinio/pull/560/files#diff-3ea7fa44784ec45a11489f5d0b1d94d2ce86ea59f9e971eb44944eed3be1b080R161

Reply:
A fix would be to not allow the user to specify the magic domain except only during installation. This value will have to be stored somewhere in order to be used in post-installation code.

This becomes a bit complicated and I start to think that we should better spend time working on stories that solve the real issue (https://github.com/epinio/epinio/pull/560#issuecomment-871121956). No domain should be special in the code and no decisions should be made based on the domain. Our defaults just have work together (e.g. default domain when one is not set: `1.2.3.4.omg.howdoi.website`, default issuer when one is not defined: `CA issuer`).

Reply:
Yes, I agree, there should be no special domain. Instead we should allow the user to configure features.

It was interesting to see how far omg.whowi.website spread across the code.
CI and DNS is working again, we no longer need this workaround, so I'd close this issue.